From 514a9a1ab0d9de6bffb4398b6202b100401263b4 Mon Sep 17 00:00:00 2001
From: Jeremy Kerr <jk@ozlabs.org>
Date: Thu, 15 Aug 2019 11:25:14 +0800
Subject: [PATCH 1/9] pb-console: start shell as a login shell

The environment that pb-console starts is minimal, as we have likely
been run by udev.

Because we're starting a shell, we want the proper profile set up, so
run as a login shell.

This assumes the shell accepts -l, but we already have an assumption for
supporting -m (as /bin/ash does).

Signed-off-by: Jeremy Kerr <jk@ozlabs.org>
---
 utils/pb-console | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/pb-console b/utils/pb-console
index ab5eaaa..8245b33 100644
--- a/utils/pb-console
+++ b/utils/pb-console
@@ -141,5 +141,5 @@ while :
 do
 	$ui $verbose_opt
 	reset
-	$shell -m
+	$shell -ml
 done
-- 
2.17.1

